projects/plugins/social/src/js/components/header/index.js

```diff
@@ -63,6 +66,12 @@ const Header = () => {
 								loading: null === sharesCount,
 								value: formatter.format( sharesCount ),
 							},
+							{
+								icon: <Icon icon={ postList } />,
+								label: __( 'Posted this month', 'jetpack-social' ),
+								loading: null === postsCount,
+								value: formatter.format( postsCount ),
```

Comment:
I'm coming to this late, but I was wondering why you went in this direction to format numbers. Maybe it would be simpler to rely on `numberFormat`, also available in the Components package which you use here. It would avoid some duplication maybe.

Reply:
@jeherve I don't think we knew that existed! I can submit a follow-up PR to use that rather than our own implementation, as well as the textual change @pablinos suggested.